src/clj/qbits/jet/client/http.clj

```diff
@@ -10,24 +10,35 @@
     [qbits.jet.client.cookies :as cookies]
     [qbits.jet.util :as util])
   (:import
+    (clojure.lang
```

Comment:
Please avoid cleanup activity (sorting) at the same time we're making substantive changes.

Reply:
Will try and resist the urge :(